fvm/src/gas/price_list.rs

```diff
@@ -241,9 +238,14 @@ lazy_static! {
         .copied()
         .collect(),
 
-        // TODO(#1277): Implement this first before benchmarking.
         // TODO(#1384): Reprice
-        get_randomness_seed: Gas::new(21000),
+        get_randomness_seed: Gas::new(36000),
+        get_tipset_cid_seed: Gas::new(36000),
```

Comment:
Seed?

Reply:
Also, given that the charges are the same, maybe just roll it into the lookback cost?

Reply:
Sure. I thought I'd keep it granular in case we want to diverge later, but no preference.

Reply:
Yeah, we may need to do that later. But we have a lot of "maybe we'll use this later" things in the price list so I want to avoid adding too many more.